meta-mbl-distro/scripts/create-update-payload

```diff
@@ -179,12 +180,23 @@ def create_payload_tar_archive(tar_archive_name, dir_path):
     :param tar_archive_name str: Path to the tar archive.
     :param dir_path Path: Path to the directory of the files to archive.
     """
+    files_for_tar = []
+    for fname in dir_path.iterdir():
+        gzip_fname = "{}.gz".format(
+            list(fname_var_map.keys())[
+                list(fname_var_map.values()).index(fname.name)
```

Comment:
This confuses me... IUUC you're using the dict to find a key from a value. Would it be clearer if we just had another dictionary that maps in the opposite direction:
```
{ filename : varname for varname, filename in fname_var_map.items() }
```

Can we at least have a comment about what's going on here please?

Reply:
e5be650